package.json

```diff
@@ -66,6 +67,7 @@
     "graphql-scalars": "^1.22.2",
     "graphql-subscriptions": "^2.0.0",
     "graphql-type-json": "^0.3.2",
+    "graphql-upload": "^16.0.2",
```

Comment:
@viniciusriquieri 
in a comment earlier I said that I didn't want new packages to be installed, but I can see a few of them there
all the tests regarding the mutations are still failing due to that